src/Symfony/Component/Security/Http/Authentication/AuthenticatorManager.php

```diff
@@ -56,9 +63,16 @@ public function __construct(
         private string $firewallName,
         private ?LoggerInterface $logger = null,
         private bool $eraseCredentials = true,
-        private bool $hideUserNotFoundExceptions = true,
+        string|bool $exposeSecurityErrors = self::EXPOSE_ACCOUNT_STATUS,
         private array $requiredBadges = [],
     ) {
+        if (\is_bool($exposeSecurityErrors)) {
+            trigger_deprecation('symfony/security-http', '7.3', 'Passing a boolean as "exposeSecurityErrors" parameter is deprecated, use string value instead.');
+
+            $exposeSecurityErrors = self::EXPOSE_NONE;
```

Comment:
This is wrong. Not all booleans should become `self::EXPOSE_NONE`. `true` and `false` have different meaning.